Assets/AnkrDemo/Scripts/WebGLHeaderWalletsPanel.cs

```diff
@@ -28,9 +27,8 @@ private void CreateButtons()
 		{
 			foreach (var wallet in _wallets)
 			{
-				var button = Instantiate(_buttonPrefab);
-				button.transform.SetParent(_panel.transform, false);
-				var buttonScript = button.GetComponent<HeaderWalletButton>();
+				var buttonScript = Instantiate(_buttonPrefab);
+				buttonScript.transform.SetParent(_panel.transform, false);
```

Comment:
I think you can just use Instantiate(prefab, parent) instead of setting parent explicitly, don't see a reason to use false for worldpositionstays?

Reply:
Without that argument prefabs scales for multiple times.